modules/alerting/variables.tf

```
@@ -8,8 +8,8 @@ variable "notification_channels" {
   default     = []
 }
 
-variable "notification_channels_pagerduty" {
-  description = "Email notification channel."
+variable "notification_channels_incident_io" {
```

```suggestion
variable "notification_channels_pager" {
```

lets rename it for the purpose and avoid using a service name